services/sidecar/Makefile

```diff
@@ -0,0 +1,95 @@
+#
+# TODO: under development
+#
+.DEFAULT_GOAL := help
```

Comment:
Check New Makefiles e.g. in catalog and include the base Makefile
If you can, please update the cookiecutter :-)

Reply:
the cookiecutter? but it does not have access to the common makefile folder does it?

Reply:
done for sidecar.

Reply:
> the cookiecutter? but it does not have access to the common makefile folder does it?

Aha! True. I did not think of that. We can perhaps checkout it?